src/services/transfer/__tests__/transfer-out-ehr-core.test.js

```diff
@@ -49,34 +47,34 @@ describe('transferOutEhrCore', () => {
       expect(sendCore).not.toHaveBeenCalled();
     });
 
-
     it('should validate incomplete EHR where failed to retrieve EHR URL', async () => {
+      // when
       getRegistrationRequestStatusByConversationId.mockResolvedValueOnce(null);
       getEhrCoreFromRepo.mockRejectedValueOnce(new EhrUrlNotFoundError());
+      // updateRegistrationRequestStatus.mockResolvedValueOnce(undefined);

```

Comment:
```suggestion
      // updateRegistrationRequestStatus.mockResolvedValueOnce(undefined);
```
Should this be commented out/removed?